GTHC/web
# Login redesign

## Description
Redesign the login page

## How Has This Been Tested?
- Run the app
- Go to login page
- See new login page

## Screenshots (if appropriate):
<img width="1438" alt="screen shot 2019-01-14 at 8 15 59 pm" src="https://user-images.githubusercontent.com/15840164/51151664-a3b10280-1839-11e9-8452-95517abc593e.png">

## Types of changes

- [ x] New feature (non-breaking change which adds functionality)

## Checklist:
